moped-editor/src/views/projects/projectView/ProjectTimeline.js

```diff
@@ -157,19 +155,16 @@ const ProjectTimeline = ({ refetch: refetchSummary }) => {
       .reduce(
         (obj, item) =>
           Object.assign(obj, {
-            [item.subphase_name.toLowerCase()]:
-              item.subphase_name.charAt(0).toUpperCase() +
-              item.subphase_name.slice(1),
+            [item.subphase_id]: item.subphase_name,
           }),
         {}
       );
 
   /**
-   * If new or updated phase has a current phase of true,
-   * set current phase of any other true phases to false
+   * If phaseObject has is_current_phase === true,
+   * set is_current_phase of any other true phases to false
```

Comment:
This comment makes me think that we're about to see a mutation being fired off one for each phase except for the "current" one. We'll never have enough phases for this to really be a big deal, but conceptually, ajax request in loops isn't ideal. Also, I totally recognize that this may be way outside the scope of your PR.


I feel like since only one phase is ever allowed to be the "current" one, the actual phase that is current is an intrinsic property of the project, not of the individual phases. Above, I see you're adding a field to track which phase is current in the project table, but I'm confused about what is going on here in this loop.

Reply:
This is an interesting point. Seems like your suggestion would be a bigger schema refactor. Maybe worth documenting in a new issue as tech debt.

Reply:
I was only updating the comment description for this function; I didn't change any of the functionality that existed before. 

If the new phase (or edited phase) is marked as current in the UI, then this function loops through all the phases for the project and if one is already marked as "is_current_phase", it sets it to false.